lint-staged.config.cjs

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  '*.{js,mjs,cjs,vue}': [
+    'eslint --fix',
+    'vitest related --run',
+  ],
+  '**/dialtone-icons/src/keywords-icons.json': (filenames) => 'nx run dialtone-icons:build --skip-nx-cache',
```

Comment:
Not sure this is going to work, does this automatically add the changes to the commit after the command is run?

- I think we could detect changes on `packages/dialtone-icons/src/*.json` instead to include icons and illustrations json files.

- Why the command is receiving a function parameter instead of being just an array like the others? Whe're not using the parameter anyways.